CONTRIBUTING.rst

```diff
@@ -380,6 +380,56 @@ pull requests from other contributors yourself after having reviewed
 them.
 
 
+Merge/squash guidelines
+-----------------------
+
+When a PR is approved and ready to be integrated to the `main` branch, one has the option to *merge* the commits unchanged, or *squash* all the commits into a single commit.
+
+Here is some guidelines on how to proceeed, based on the state of the PR commit history:
+
+1. Miscellaneous commits:
```

Comment:
(withing the same PR?)

Reply:
Yes, I did change the introductory phrase just before the example in an attempt to make it clearer.

Reply:
okay, fair